src/app/courses/cousers-icon.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'planet-course-icon',
+  template: `
+  <ng-container [ngSwitch]="icon">
+    <span *ngIf="step?.exam?.questions.length" i18n-matTooltip matTooltip="The test has question">
+      <mat-icon *ngSwitchCase="'assignment'">assignment</mat-icon>
```

Comment:
`*ngSwitchCase` must be used on a child element of the `[ngSwitch]`.  This does nothing as it is written.